### ISSUE TITLE
Move China 1952 data imputation into separate script and document the how

### ISSUE THREAD

**Issue by hlapp** [jennybc/gapminder#6]
Data imputation seems like a fairly fundamental manipulation to a dataset to me, so where and how this is done should be as transparent as possible. Right now this step is lumped together with the standardization on every 5 years in [07_filter-every-five-years](https://github.com/jennybc/gapminder/blob/master/data-raw/07_filter-every-five-years.r#L58-L125), and the name of the file doesn't offer any suggestion that this is the place where to find it.

Also, the imputations steps undertaken, while evident from studying the code, don't seem to be summarized somewhere in plain English (although I may well be missing where they are). I'd suggest adding that at the top of a stand-alone script for accomplishing this step.

**Comment by jennybc** [comment]
Yes, this is a good idea. I plan to revisit all the cleaning scripts and make them "spinnable". I'll split this off at that time.
